api_tests/collection_submission_actions/views/test_collection_submissions_actions_list.py

```diff
@@ -206,6 +206,21 @@ def test_POST_reject__writes_action_and_advances_state(self, app, collection_sub
         assert action.to_state == CollectionSubmissionStates.REJECTED
         assert collection_submission.state is CollectionSubmissionStates.REJECTED
 
+    def test_POST_cancel__writes_action_and_advances_state(self, app, collection_submission, node):
```

Comment:
Any error cases we should be testing? CANCEL from unsupported states? Permissions tests for CANCEL, too, plz.